examples/initiator_group.go

```diff
@@ -92,7 +92,7 @@ func main() {
 	// get igroup by name
 	igroup, err = igroupSvc.GetInitiatorGroupByName("sdkigroup")
 	if err != nil {
-		fmt.Printf("Failed to get igroup by name, err ", err)
+		fmt.Printf("Failed to get igroup by name, err: %v", err)
```

Comment:
Since Printf doesn't include a trailing newline, it is common to add \n to the end of the message.